library/sha3.c

```diff
@@ -280,7 +283,11 @@ int mbedtls_sha3_finish(mbedtls_sha3_context *ctx,
         }
     }
 
-    return 0;
+    ret = 0;
+
+exit:
+    mbedtls_platform_zeroize(ctx, sizeof(mbedtls_sha3_context));
```

Comment:
How about calling `mbedtls_sha3_free` instead, for more robustness?

Reply:
Not convinced it adds a whole lot of robustness, but it does save 2 bytes.